components/omega/src/ocn/Eos.cpp

```diff
@@ -218,6 +270,18 @@ void Eos::defineFields() {
                      NDims,     // Number of dimensions
                      DimNames   // Dimension names
        );
+   /// Create and register the BruntVaisalaFreq field
+   auto BruntVaisalaFreqField =
+       Field::create(BruntVaisalaFreqFldName,                     // Field name
+                     "Brunt-Vaisala frequency squared",           // Long Name
+                     "s-2",                                       // Units
+                     "sea_water_brunt_vaisala_frequency_squared", // CF-ish Name
+                     0.0,       // Min valid value
```

Comment:
BVSq can be negative so min val should be < 0

Reply:
Good catch, thanks! I will adjust this.